packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -190,7 +218,8 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
     this.addTypeToResourceReferences(def.name.value, this.rules);
     // turn rules into roles and add into acm and roleMap
     this.convertRulesToRoles(acm, this.rules, isJoinType, undefined, undefined, context);
-    this.modelDirectiveConfig.set(typeName, getModelConfig(modelDirective, typeName, context.featureFlags, context.isProjectUsingDataStore()));
+    this.modelDirectiveConfig.set(typeName,
+      getModelConfig(modelDirective, typeName, context.featureFlags, context.isProjectUsingDataStore()));
```

Comment:
Not a blocker, but just a n.p. There are a few formatting/newline changes in this CR that distract from the fact that you're basically just changing the error string.

Reply:
Yeap. The thing was I would get a warning every time I tried to `yarn commit`. It pointed out those es-lint errors and `yarn lint-fix` wouldn't fix them